doc/content/devices/models/mkr1310/temp.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Sending Temperature with MKR WAN 1310"
+description: ""
+weight:
+---
+
+In this section you are going to try an example project where you will connect a Temperature and Humidity sensor to the MKR WAN 1310 to send the data over LoRaWAN and then decode it on {{% tts %}}.
+
+<!--more-->
+
+{{< figure src="../fritzing.png" alt="Wiring mockup of the DHT11 connected to the arduino" class="float plain" width="90%">}}
+
+## Circuit
+
+For this project you are going to be needing:
+- **DHT11**
+- Some **Jumper Wires**
+- **Breadboard**
+
+Once you have the components, wire it up like so:
+- **Arduino 5V** -> **Plus DHT11**
+- **Arduino GND** -> **Minus DHT11**
+- **Arduino Pin 6** -> **Signal DHT11**
+
+Illustration on the right displays the wiring as well.
+
+{{< note "Double check your own sensor's pinout, it can vary between modules. If you are using a bare DHT11, you might need to use a resistor." />}}
+
+## Creating the code
```

Comment:
```suggestion
## Programming the MKR 1310
```

Reply:
I think this whole section is overkill. It would be better to just paste the complete code and eventually add comments where needed

Reply:
Alright fair. I just thought it'd be good to explain to the users what the code is doing exactly.